### PR TITLE
value keyword: mention indexer usage

### LINKED ISSUE #6487: one use of value keyword missing
The contextual keyword "value" can be used in the set accessor of indexers too.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c4e70b29-7da5-0604-0aa7-b13800a3cd9d
* Version Independent ID: 131d33af-fbc4-20e9-46bb-0a92e6254dbb
* Content: [value (C# Reference)](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/value)
* Content Source: [docs/csharp/language-reference/keywords/value.md](https://github.com/dotnet/docs/blob/master/docs/csharp/language-reference/keywords/value.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by pkulikov**
Fixes #6487

### CODE REVIEW

File: docs/csharp/language-reference/keywords/value.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ ms.assetid: c99d6468-687f-4a46-89b4-a95e1b00bf6d
...
+The contextual keyword `value` is used in the `set` accessor in [property](../../programming-guide/classes-and-structs/properties.md) and [indexer](../../programming-guide/indexers/index.md) declarations. It is similar to an input parameter on a method. The word `value` references the value that client code is attempting to assign to the property or indexer. In the following example, `MyDerivedClass` has a property called `Name` that uses the `value` parameter to assign a new string to the backing field `name`. From the point of view of client code, the operation is written as a simple assignment.
 
 [!code-csharp[csrefKeywordsModifiers#26](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsModifiers/CS/csrefKeywordsModifiers.cs#26)]
 
-For more information about the use of `value`, see [Properties](../../programming-guide/classes-and-structs/properties.md).
+For more information, see the [Properties](../../programming-guide/classes-and-structs/properties.md) or [Indexeres](../../programming-guide/indexers/index.md) article.
```
Comment (Youssef1313) [inline docs/csharp/language-reference/keywords/value.md]:
```suggestion
For more information, see the [Properties](../../programming-guide/classes-and-structs/properties.md) and [Indexeres](../../programming-guide/indexers/index.md) articles.
```

File: docs/csharp/language-reference/keywords/value.md [hunk 2]
```diff
@@ -10,11 +10,11 @@ ms.assetid: c99d6468-687f-4a46-89b4-a95e1b00bf6d
 ---
 # value (C# Reference)
 
-The contextual keyword `value` is used in the set accessor in ordinary property declarations. It is similar to an input parameter on a method. The word `value` references the value that client code is attempting to assign to the property. In the following example, `MyDerivedClass` has a property called `Name` that uses the `value` parameter to assign a new string to the backing field `name`. From the point of view of client code, the operation is written as a simple assignment.
+The contextual keyword `value` is used in the `set` accessor in [property](../../programming-guide/classes-and-structs/properties.md) and [indexer](../../programming-guide/indexers/index.md) declarations. It is similar to an input parameter on a method. The word `value` references the value that client code is attempting to assign to the property or indexer. In the following example, `MyDerivedClass` has a property called `Name` that uses the `value` parameter to assign a new string to the backing field `name`. From the point of view of client code, the operation is written as a simple assignment.
```
Comment (Youssef1313) [inline docs/csharp/language-reference/keywords/value.md]:
```suggestion
The contextual keyword `value` is used in the `set` accessor in [property](../../programming-guide/classes-and-structs/properties.md) and [indexer](../../programming-guide/indexers/index.md) declarations. It is similar to an input parameter of a method. The word `value` references the value that client code is attempting to assign to the property or indexer. In the following example, `MyDerivedClass` has a property called `Name` that uses the `value` parameter to assign a new string to the backing field `name`. From the point of view of client code, the operation is written as a simple assignment.
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

File: docs/csharp/language-reference/keywords/value.md (hunk 2 above)
Comment (pkulikov) [reply docs/csharp/language-reference/keywords/value.md]: @Youssef1313 I can't spot the change. Could you please point me to the change you're suggesting?

Comment (Youssef1313) [reply docs/csharp/language-reference/keywords/value.md]: parameter ((on)) a method -> of

Comment (pkulikov) [reply docs/csharp/language-reference/keywords/value.md]: @Youssef1313 indeed, thanks!

**Review by BillWagner — APPROVED** [review]
This LGTM @pkulikov 

I'll :shipit: now.

**Pull request merged**